Assets/Scripts/Game/GameManager.cs

```diff
@@ -59,23 +61,43 @@ private IEnumerator HandleWallsDangerousness()
             {
                 yield return null;
```

Comment:
Do we really need to skip the first frame?

Reply:
No, it is probably redundant